azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/middleware/sdk/ChatServiceListener.kt

```diff
@@ -101,10 +112,17 @@ internal class ChatServiceListener(
             is ParticipantTimestampInfoModel -> {
                 when (it.eventType) {
                     ChatEventType.TYPING_INDICATOR_RECEIVED -> {
-                        dispatch(ParticipantAction.TypingIndicatorReceived(message = it.infoModel))
+                        val infoModel = it.infoModel
+                        dispatch(ParticipantAction.AddParticipantTyping(infoModel = infoModel))
```

Comment:
Same here, AddParticipantTyping probably should just take a UID of user.

Reply:
We need both id+timestamp to identify unique notification